### PR TITLE
menu: new "Add Fake Airplay" and "Add Fake Chromecast" items

### LINKED ISSUE #21: Need way to simulate chromecast/airplay devices (for development)
I want a way to see the icons when I'm working from a coffee shop.

### PR DESCRIPTION
**Pull request by feross**
Fixes #21

### CODE REVIEW

File: renderer/index.js [hunk 1]
```diff
@@ -191,6 +192,13 @@ electron.ipcRenderer.on('fullscreenChanged', function (e, isFullScreen) {
   update()
 })
 
+electron.ipcRenderer.on('addFakeDevice', function (e, device) {
+  var player = new EventEmitter()
+  player.play = (networkURL) => window.alert(networkURL)
```
Comment (ngoldman) [inline renderer/index.js]: maybe a console.log instead? alerts get annoying pretty quickly

Comment (feross) [inline renderer/index.js]: sure thing

**Comment by ngoldman** [comment]
LGTM, just a note about using `window.alert()`, the clippy of the browser DOM.

**Comment by feross** [comment]
fixed it already :)

**Pull request merged**